src/main/java/com/sse/ooseproject/models/Room.java

```diff
@@ -0,0 +1,91 @@
+package com.sse.ooseproject.models;
+
+
+import jakarta.persistence.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "room")
+public class Room {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    private String number;
+    private int seats;
+    private boolean isAuditorium;
+    @OneToMany
```

Comment:
Hier könnte man doch schon (mappedBy = "room") hinzufügen, oder? In der RoomOccupancy.java ist ja auch schon die entsprechende Spalte für den Join angegeben?